usaspending_api/database_scripts/tests/test_matview_sql_generator.py

```diff
@@ -1,8 +1,11 @@
 import pytest
 
 from django.db import connection
+from collections import OrderedDict
 from usaspending_api.common.helpers.generic_helper import generate_matviews
-from usaspending_api.common.matview_manager import MATERIALIZED_VIEWS
+from usaspending_api.common.matview_manager import MATERIALIZED_VIEWS, CHUNKED_MATERIALIZED_VIEWS
+
+ALL_MATVIEWS = OrderedDict(list(MATERIALIZED_VIEWS.items()) + list(CHUNKED_MATERIALIZED_VIEWS.items()))
```

Comment:
Dictionaries already preserve order in Python 3.6+ (happy side-effect from other changes in 3.6, added to language specification for 3.7 and forward)

Reply:
Perhaps this will satisfy the combining of dictionaries.

```suggestion
ALL_MATVIEWS = {**MATERIALIZED_VIEWS, **CHUNKED_MATERIALIZED_VIEWS}
```